hal/architecture/Linux/drivers/core/interrupt.h

```diff
@@ -24,6 +24,9 @@
 
 #include <stdint.h>
 
+// Ajout RRO
+#include <gpiod.h>
```

Comment:
Do you know if this is installed by default on Raspberry Pi OS? If it is not, we should probably document how to install the dependency.